splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/actions/AlterTableConstantOperationIT.java

```diff
@@ -699,13 +694,19 @@ private void testAlterAddColumnWithEmptyDefault(boolean insertBeforeAlter, Strin
     @Test
     public void testAlterAddColumnInEmptyTableWithEmptyDefault() throws Exception {
         String tableName = "test_alter_add_column_in_empty_table_with_empty_default";
-        testAlterAddColumnWithEmptyDefault(false, tableName);
+        testAlterAddColumnWithEmptyDefault(false, tableName, false, false);
     }
 
     @Test
     public void testAlterAddColumnInFilledTableWithEmptyDefault() throws Exception {
         String tableName = "test_alter_add_column_in_filled_table_with_empty_default";
-        testAlterAddColumnWithEmptyDefault(true, tableName);
+        testAlterAddColumnWithEmptyDefault(true, tableName, false, false);
+    }
+
+    @Test
+    public void testAlterAddColumnInEmptyTableThenSetAnEmptyDefault() throws Exception {
```

Comment:
@dgomezferro this test is doing that:
setDefaultSeparately = true, so we'll create the column, then alter the default
insertBeforeAlter = false, so we'll alter the table, then insert

Reply:
I see, thanks!